test/main/faqs/index.spec.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import MessengerMessageUs from 'react-messenger-message-us';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+import Banner from '../../../src/main/common/Banner';
+import FaqsComponent from '../../../src/main/faqs';
+import FaqsAccordions from '../../../src/main/faqs/FaqsAccordions';
+import '../../../src/main/faqs/index';
+
+describe('FaqsComponent', () => {
+    it('should render banner, accordions, messenger message plugin and phone number', () => {
+        const wrapper = shallow(<FaqsComponent />);
+        const expectedBanner = (
+            <Banner
+                title="FAQs"
+                imagePath="/static/images/about/banner-background.png"
+            />
+        )
+        const expectedAccordions = (<FaqsAccordions />);
```

Comment:
No need for `(<FaqsAccordions />)`

you can use

`expect(wrapper.contains(FaqsAccordions)).to.equal(true);`